### ISSUE TITLE
Supervised learning portion of the project

### ISSUE THREAD

**Issue by SBuxton-IDM** [SBuxton-IDM/EquityMilestone#6]
This issue is meant to organize several different issues relating to the supervised learning portion of the project. As the project progresses I may add more issues/checkboxes as it becomes clearer what the specific tasks are. In other words this might be more of an outline in the beginning :)

- [ ] Break our 5 feature dataset into a test, training, and validation set. Eventually I think we will use k-fold validation but this is a good start
- [ ] Train our three types of model using these data
- [ ] Show a summary of the results using something like the summary function (https://www.geeksforgeeks.org/how-to-get-regression-model-summary-from-scikit-learn/) or a stargazer table (https://pypi.org/project/stargazer/)
- [ ] Based on the summary statistics, we will need to pick a model and tune the parameters. To show that we are doing this effectively, we should isolate a parameter, set it to a range of values, and choose an evaluation metric (like accuracy), and show how this metric changes over our chosen values. So for example, we might have one line graph for "number of trees" where the x-axis is the number of trees and the y-axis is accuracy.
- [ ] With our model tuned, we should have some sort of summary table to show the final results.
